db/db.js

```diff
@@ -32,6 +32,19 @@ const insertTopic = (causeObj, cb) => {
     });
 };
 
+const getTopic = (topicName, cb) => {
+    topicName = topicName.replace(`'`,`''`)
+    pool.query(`select * from topics where lower(topic_name) like lower('%${topicName}%')`, (err, data) => {
+        if (err) {
+            cb(err);
+        } else {
+            cb(null,data);
+        }
+    });
+}
 
 module.exports.getAll = getAll
+
+module.exports.getTopic = getTopic
+
 module.exports.insertTopic = insertTopic
```

Comment:
Can all of these get put together?
module.exports = {getAll, getTopic, insertTopic}?

Reply:
Completed.